src/templates/next/components/internal/ContentPageHeader/ContentPageHeader.tsx

```diff
@@ -3,25 +3,28 @@ import Button from "../../complex/Button"
 import Breadcrumb from "../Breadcrumb"
 import BaseParagraph from "../BaseParagraph"
 import { Paragraph } from "../../../typography/Paragraph"
+import { getFormattedDate } from "~/utils/getFormattedDate"
```

Comment:
omega nit: can we import only from `~/utils`, so we export this inside `~/utils/index.ts`

Reply:
Fixed in https://github.com/isomerpages/isomer-components/pull/88/commits/78358336580097e5134b964a0f4942822891c994!